src/cryptoadvance/specter/controller.py

```diff
@@ -399,6 +405,7 @@ def general_settings():
                     wallet_obj = app.specter.wallet_manager.get_by_alias(
                         wallet['alias']
                     )
+                    wallet_obj.import_labels(wallet['labels'])
```

Comment:
What if the old backup is imported? 'labels' will not be there then.
You can use `wallet.get("labels", [])` to get empty if not found.

Reply:
Right, changed it.